### PR TITLE
Remove course score callbacks and archive students from admin

### PR DESCRIPTION
**Pull request by nathanstitt**
Needed so an LMS account can be un-associated with a tutor student and re-launched to join a different account

goes along with https://github.com/openstax/accounts/pull/644

### CODE REVIEW

File: app/controllers/admin/courses_controller.rb [hunk 1]
```diff
@@ -38,6 +38,18 @@ def unpair_lms
     redirect_to admin_courses_path
   end
 
+  def drop_student
+    student = CourseMembership::Models::Student.find(params[:id])
```
Comment (jpslav) [inline app/controllers/admin/courses_controller.rb]: isn't this `:id` the ID of the course?

File: app/views/layouts/_bootstrap_tab_activate_on_load.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<script>
+  $(document).ready(function() {
+      var hash = window.location.hash;
+      hash && $('.nav-tabs a[role="tab"][href="' + hash + '"]').tab('show');
+      $('ul.nav.nav-pills a').click(function (e) {
+        $(this).tab('show');
+        var scrollmem = $('body').scrollTop();
+        window.location.hash = this.hash;
+      });
```
Comment (jpslav) [inline app/views/layouts/_bootstrap_tab_activate_on_load.html.erb]: it is like you know javascript or something

File: app/controllers/admin/courses_controller.rb (hunk 1 above)
Comment (jpslav) [reply app/controllers/admin/courses_controller.rb]: per the routes this is the ID of the course, but I see you are explicitly passing in the student ID.  we probably need to make the routes consistent?

File: app/views/layouts/_bootstrap_tab_activate_on_load.html.erb (hunk 2 above)
Comment (nathanstitt) [reply app/views/layouts/_bootstrap_tab_activate_on_load.html.erb]: heh, yeah I got tired of loading the page, then switching tabs

**Pull request merged**